pkg/internal/cluster/providers/docker/provision.go

```diff
@@ -242,7 +244,11 @@ func getSubnets(networkName string) ([]string, error) {
 func generateMountBindings(mounts ...cri.Mount) []string {
 	args := make([]string, 0, len(mounts))
 	for _, m := range mounts {
-		bind := fmt.Sprintf("%s:%s", m.HostPath, m.ContainerPath)
+		absHostPath, err := filepath.Abs(m.HostPath)
+		if err != nil {
+			globals.GetLogger().Errorf("Couldn't determine absolute path for extraMounts", err)
```

Comment:
let's do this at a layer above and return the error, unless this is expected.

Reply:
it's not expected. In this case we proceed merrily to create a directory anyway. I can change it propogate the error up I guess.

Reply:
er rephrasing: this is a common thing we need to resolve anyhow, this should be done outside of the provider.

Reply:
Are you referring to mutating hostpath? Or error handling?

Reply:
Are you referring to mutating hostpath? Or error handling?

Reply:
Alright I moved this to validate.go as that seems the easiest. It will now error out if you specify a hostPath that doesn't exist :+1: